spark/src/main/java/org/apache/zeppelin/spark/SparkSqlInterpreter.java

```diff
@@ -129,8 +116,15 @@ public InterpreterResult interpret(String st, InterpreterContext context) {
       sc.setLocalProperty("spark.scheduler.pool", null);
     }
 
+    Object rdd = null;
+    try {
+      Method sqlMethod = sqlc.getClass().getMethod("sql", String.class);
+      rdd = sqlMethod.invoke(sqlc, st);
```

Comment:
Would you elaborate on why we would need to do this? Curious -  isn't SQLContext always has the sql() method?